payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SharedDataSpec.kt

```diff
@@ -16,7 +16,7 @@ data class SharedDataSpec(
     // If a form is empty, it must still have an EmptyFormSpec
```

Comment:
Can you update or remove this comment? It's not relevant anymore

Reply:
Good call. Thanks!